erpnext/hr/doctype/leave_control_panel/leave_control_panel.json

```diff
@@ -2,6 +2,7 @@
  "allow_copy": 1, 
  "allow_import": 0, 
  "allow_rename": 0, 
+ "beta": 0, 
```

Comment:
Use master branch of `frappe` so that `beta`, `track_seen`, etc. does not appear in hotfix branch